src/net/blackscarx/betterchairs/UpdateManager.java

```diff
@@ -24,10 +24,10 @@
 
     public String getLastSpigotVersion() {
         try {
-            HttpURLConnection con = (HttpURLConnection) new URL("http://dev.blackscarx.com/betterchairsversion.txt").openConnection();
+            HttpURLConnection con = (HttpURLConnection) new URL("http://BlackScarx.github.io/BetterChairs/version.txt").openConnection();
```

Comment:
Set to https://jlsjonas.github.io/BetterChairs/version.txt in the compiled version.